cli/cli.go

```diff
@@ -117,7 +117,7 @@ func NewCommand(logdest logger.Logger) *Command {
 		workingDir: newWorkingDir(),
 
 		Stdin:  Stdin,
-		Stdout: Stdout,
+		stdout: Stdout,
```

Comment:
Why does `stdout` have to be lowercase?

Reply:
I needed to add a setter method for `stdout`, because it has to do more than just set the value. We don't currently need that for `Stdin` or `Stderr` so I didn't bother changing those.

I suppose if we want to be consistent/proactive, we could make setter methods for those as well. I can do that.

Reply:
@adrwalker I pushed a commit which adds setters for those.